docs/src/matrix.md

```diff
@@ -134,15 +155,6 @@ d = det_given_divisor(A, c)
 pseudo_inv(::ZZMatrix)
 ```
 
-**Examples**
-
-```julia
-S = matrix_space(ZZ, 3, 3)
-
-A = S([1 0 1; 2 3 1; 5 6 7])
-
-B, d = pseudo_inv(A)
```

Comment:
I was confused why this example is gone, but then I saw that you moved it into the `pseudo_inv` docstring -- very good